CriticalMass/Date+Additions.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  CriticalMaps
+
+import Foundation
+
+public extension Date {
+    static var yesterday: Date { Date(timeInterval: -86400, since: .now) }
```

Comment:
I'm always scared of adding seconds to a Date because a Date might contain leap seconds.. 
Better use Calendar for that https://developer.apple.com/documentation/foundation/calendar/2293676-date

Reply:
Ah. So you mean this could end up in not being yesterday. Haven’t thought of that. But it is just wrapping what was in the tests before. But might be a good idea to rework to Calendar

Reply:
Updated to Calendar and added some UnitTests